### PR TITLE
clean code job options

### PR DESCRIPTION
**Pull request by ayocodingit**
### Overview

config job git
`  delay: 10000,
  attempts: 5,
  priority: 1,
  removeOnComplete: true,
  backoff: 5000,
  timeout: 30000`
config elastic
`delay: 10000,
  priority: 2,
  removeOnComplete: true,
  backoff: 5000,
  timeout: 10000`

## Evidence
title: clean code job options generate report webhook
project: Digiteam
participants: @firmanalamsyah580 @budiaramdhanrindi @sandisunandar @novansyaah

cc: @jabardigitalservice/jds-backend

### CODE REVIEW

**Review by rindibudiaramdhan — APPROVED** [review]

**Comment by rindibudiaramdhan** [comment]
Nice job @ayocodingit more readable and easy to maintain if change needed

**Review by rachadiannovansyah — APPROVED** [review]
saving more rows by used const, lgtm!

File: server.js [hunk 1]
```diff
@@ -6,6 +6,15 @@ import connectQueue from './utils/connectQueue.js'
 import isMerged from './utils/isMerged.js'
 dotEnv.config()
 
+const jobOptions = {
```
Comment (sandisunandar99) [inline server.js:9]: mending di simpen di utils aja supaya setiap code confifg semua ada di folder yg sama,, kaya koneksi koneksi db, token aplikasi 3rd, dsb

Comment (ayocodingit) [reply server.js]: in here dude @sandisunandar99 :+1: 
9a13115

File: utils/jobOptions.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+const gitOptions = {
```
Comment (sandisunandar99) [inline utils/jobOptions.js:1]: Nice good . L.G.T.M

**Review by sandisunandar99 — APPROVED** [review]

File: server.js (hunk 1 above)
Comment (sandisunandar99) [reply server.js]: okayy

**Pull request merged**